playbooks/roles/capsule/tasks/devel_install.yml

```diff
@@ -0,0 +1,19 @@
+- name: 'start rails server'
+  command: /home/vagrant/.rvm/bin/rvm ruby-2.2.4 do rails s
```

Comment:
@ehelms I have to recall my findings when I wrote this, but I think the rvm and ansible relationship can be tricky and this was a workaround I found to use an rvm environment. I'll investigate further today. There is a [plugin](https://github.com/rvm/rvm1-ansible) that we may find useful